gateway/src/apicast/module.lua

```diff
@@ -39,7 +39,7 @@ local name = env.value('APICAST_MODULE') or 'apicast.policy.apicast'
 local ok, mod = prequire(name)
 
 if ok and mod then
-  if type(mod) == 'table' then
+  if type(mod) == 'table' or true then
```

Comment:
This is always true.

Reply:
I know. Needed to do this temporarily because `type(mod)` returns `userdata`.